common/src/main/java/earth/terrarium/cadmus/common/claims/ClaimApiImpl.java

```diff
@@ -173,24 +176,22 @@ public boolean canInteractWithEntity(Level level, Entity entity, @NotNull Player
 
     @Override
     public boolean canDamageEntity(Level level, Entity entity, UUID player) {
+        if (entity.getType().is(Cadmus.ALLOWS_CLAIM_DAMAGE_ENTITIES)) {
+            return true;
+        }
         return canAccess(level, entity.blockPosition(), player, CadmusAutoCompletes.ENTITY_DAMAGE, ModGameRules.RULE_CLAIMED_DAMAGE_ENTITIES,
             (id, server) -> {
-                if (entity.getType().getCategory() == MobCategory.CREATURE
-                    || entity.getType().getCategory() == MobCategory.AMBIENT
-                    || entity.getType().getCategory() == MobCategory.AXOLOTLS
-                    || entity.getType().getCategory() == MobCategory.UNDERGROUND_WATER_CREATURE
-                    || entity.getType().getCategory() == MobCategory.WATER_AMBIENT
-                    || entity instanceof Animal) {
-                    return AdminClaimHandler.getBooleanFlag(server, id, ModFlags.CREATURE_DAMAGE);
+                if (entity instanceof Player) {
+                    return AdminClaimHandler.getBooleanFlag(server, id, ModFlags.PVP);
                 }
 
-                if (entity.getType().getCategory() == MobCategory.MONSTER || entity instanceof Monster) {
+                if (entity.getType().getCategory() == MobCategory.MISC) {
+                    return AdminClaimHandler.getBooleanFlag(server, id, ModFlags.ENTITY_DAMAGE);
+                } else if (entity.getType().getCategory().isFriendly()) {
+                    return AdminClaimHandler.getBooleanFlag(server, id, ModFlags.CREATURE_DAMAGE);
+                } else {
                     return AdminClaimHandler.getBooleanFlag(server, id, ModFlags.MONSTER_DAMAGE);
```

Comment:
Change the order so that it checks living, then monster, and else normal entity damage